pkg/resources/mutation/cluster/provisioningcluster.go

```diff
@@ -32,13 +30,5 @@ func (m *mutator) Admit(response *webhook.Response, request *webhook.Request) er
 		return err
 	}
 
-	newCluster := cluster.DeepCopy()
-
-	if newCluster.Annotations == nil {
-		newCluster.Annotations = make(map[string]string)
-	}
-
-	newCluster.Annotations[auth.CreatorIDAnn] = request.UserInfo.Username
-
-	return patch.CreatePatch(cluster, newCluster, response)
+	return mutation.SetCreatorIDAnnotation(request, response, cluster, cluster.DeepCopy())
```

Comment:
Thought: may want to maintain that null check for `cluster.DeepCopy().Annotations()` in the `SetCreatorAnnotation` function is so you don't risk a regression